blocks/test/full-content.js

```diff
@@ -165,7 +165,9 @@ describe( 'full post content fixture', () => {
 				) );
 			}
 
-			const serializedActual = serialize( blocksActual );
+			// `serialize` doesn't have a trailing newline, but the fixture
+			// files should.
+			const serializedActual = serialize( blocksActual ) + '\n';
```

Comment:
Update when the time comes; require an exact match so that we know if this behavior changes.

I think the current behavior is correct for `post_content`, FWIW.